extensions/exposition/features/routes.feature

```diff
@@ -87,3 +87,40 @@ Feature: Routes
 
       Hello
       """
+
+  Scenario: Routes with naming conflicts
+    Given the Gateway is running
+    And the `users` is running with the following manifest:
+      """yaml
+      exposition:
+        /:
+          POST: create
+      """
+    And the `users.properties` is running with the following manifest:
```

Comment:
I may be missing some context here, but I can't see "naming conflict" there. I would say there is some intersection / common path / prefix / etc.

Reply:
Exposition doesn't use paths or prefixes for routing, it create a tree of Nodes. When processing a request, its `path` is split into segments which are then used to dive down the tree.

Tree and its routing is documented [here](https://github.com/toa-io/toa/blob/fix/exposition-routing-conflicts/extensions/exposition/documentation/tree.md) and the essential part of implementation is [here](https://github.com/toa-io/toa/blob/e2a0937ea10f768792ca0e4292c2c21256ba10c7/extensions/exposition/source/RTD/Route.ts#L22).